build/server-image/Dockerfile

```diff
@@ -14,10 +14,9 @@
 
 # Dockerfile used for the server images.
 
-ARG BASE_IMAGE_REGISTRY
-ARG BASE_IMAGE_NAME
+ARG BASE_IMAGE
```

Comment:
suggest using BASEIMAGE as it was prior to https://github.com/kubernetes/kubernetes/pull/99379/files to make release-over-release diffs on these files highlight actual changes

Reply:
done